lib/compat/wordpress-5.9/template-parts.php

```diff
@@ -0,0 +1,57 @@
+<?php
+/**
+ * Temporary compatibility shims for features present in Gutenberg.
+ * This file should be removed when WordPress 5.9.0 becomes the lowest
+ * supported version by this plugin.
+ *
+ * @package gutenberg
+ */
+
+if ( ! function_exists( 'get_allowed_block_template_part_areas' ) ) {
+	/**
+	 * Returns a filtered list of allowed area values for template parts.
+	 *
+	 * @return array The supported template part area values.
+	 */
+	function get_allowed_block_template_part_areas() {
```

Comment:
Should we move it to `block-template-utils.php` (with the other functions) to match the core patch.